src/app/data.service.ts

```diff
@@ -437,4 +437,8 @@ export class DataService {
   getBodyUrl(body: IBody): string {
     return window.location.origin + '/org/' + body.str_id;
   }
+
+  GetComplaint(uuid: string): Observable<IComplaint> {
+    return this.FireGET<IComplaint>(API.Complaint, {uuid: uuid});
+  }
```

Comment:
Is this really necessary? I can hardly imaging this function being called in more than two places.